app/templates/validation-reports/index.hbs

```diff
@@ -1 +1,54 @@
-{{outlet}}
\ No newline at end of file
+<AuDataTable
+  @content={{this.model}}
+  @fields="Conforms ShapeGraphWellFormed"
+  @noDataMessage="No reports"
+  @size={{this.size}}
+  @page={{this.page}}
+  @sort={{this.sort}} as |s|
+>
+  <s.menu as |menu|>
+    <menu.general>
+      <AuToolbar class="au-o-box">
+        <AuToolbarGroup>
+          <AuHeading @skin="2">
+            Validation reports
+          </AuHeading>
+        </AuToolbarGroup>
+        <AuToolbarGroup class="au-c-toolbar__group--center">
+          <AuDataTableTextSearch
+            @filter={{this.filter}}
```

Comment:
The controller doesn't have the needed mixin for the filtering to work. But I think the searchfield can be dropped for this table? I don't think there is anything useful to search for :smile:.